NEWS.md

```diff
@@ -1,6 +1,595 @@
+Goals of the project are: 
+* Provide MXNet JVM users automated memory management that can release native memory when there are no references to JVM objects. 
+* Provide automated memory management for both GPU and CPU memory without performance degradation.  More details can be found here: [JVM Memory Management](https://cwiki.apache.org/confluence/display/MXNET/JVM+Memory+Management)
+
+#### Topology-aware AllReduce (experimental)
+For distributed training, the `Reduce` communication patterns used by NCCL and MXNet are not optimal for small batch sizes. The `Topology-aware AllReduce` approach is based on the idea of using trees to perform the `Reduce` and `Broadcast` operations. We can use the idea of minimum spanning trees to do a binary tree `Reduce` communication pattern to improve distributed training following this paper by Wang, Li, Edo and Smola [1]. Our strategy is to use:
+
+  * a single tree (latency-optimal for small messages) to handle `Reduce` on small messages
+  * multiple trees (bandwidth-optimal for large messages) to handle `Reduce` on large messages
+
+More details can be found here: [Topology-aware AllReduce](https://cwiki.apache.org/confluence/display/MXNET/Single+machine+All+Reduce+Topology-aware+Communication)
+Note: This is an experimental feature and has known problems - see [13341](https://github.com/apache/incubator-mxnet/issues/13341). Please help to contribute to improve the robustness of the feature.
+
+#### MKLDNN backend: Graph optimization and Quantization (experimental)
+
+Two advanced features, graph optimization (operator fusion) and reduced-precision (INT8) computation, are introduced to MKLDNN backend in this release ([#12530](https://github.com/apache/incubator-mxnet/pull/12530), [#13297](https://github.com/apache/incubator-mxnet/pull/13297), [#13260](https://github.com/apache/incubator-mxnet/pull/13260)).
+These features significantly boost the inference performance on CPU (up to 4X) for a broad range of deep learning topologies. Currently, this feature is only available for the inference on the CPU platforms.
+
+##### Graph Optimization
+MKLDNN backend takes advantage of MXNet subgraph to implement the most of possible operator fusions for inference, such as Convolution + ReLU, Batch Normalization folding, etc. When using mxnet-mkl package, users can easily enable this feature by setting export MXNET_SUBGRAPH_BACKEND=MKLDNN.
+
+##### Quantization
+Performance of reduced-precision (INT8) computation is also dramatically improved after the graph optimization feature is applied on CPU Platforms. Various models are supported and can benefit from reduced-precision computation, including symbolic models, Gluon models and even custom models. Users can run most of the pre-trained models with only a few lines of commands and a new quantization script imagenet_gen_qsym_mkldnn.py. The observed accuracy loss is less than 0.5% for popular CNN networks, like ResNet-50, Inception-BN, MobileNet, etc.
```

Comment:
"Users can run most of the pre-trained models with only a few lines of commands, or they can use the new quantization script, `imagenet_gen_qsym_mkldnn.py`." I changed this to an OR because that seemed intended... but is it really an AND? 

```suggestion
Performance of reduced-precision (INT8) computation is also dramatically improved after the graph optimization feature is applied on CPU platforms. Various models are supported, and they can benefit from reduced-precision computation. This includes symbolic, Gluon, and even custom models. Users can run most of the pre-trained models with only a few lines of commands, or they can use the new quantization script, `imagenet_gen_qsym_mkldnn.py`. For popular CNN networks like ResNet-50, Inception-BN, MobileNet, etc., the observed accuracy loss is less than 0.5%.
```

Reply:
accepted